front/src/applications/operationalStudies/components/SimulationResults/SpaceTimeChart/withOSRDData.jsx

```diff
@@ -30,9 +30,8 @@ import SpaceTimeChart from './SpaceTimeChart';
  */
 const withForcedEcoAllowanceSettings = (Component) =>
   function WrapperComponent(props) {
-    const { simulation } = props;
-    let forcedEcoAllowancesSettings = [];
-    // eslint-disable-next-line react/prop-types
+    const simulation = useSelector(getPresentSimulation);
```

Comment:
I think it’s supposed to come from props to isolate the component in `storybook`, maybe check with @alexandredamiron

Reply:
oui, c'est utilisé dans le cadre d'une composition
export const ForcedEcoSpaceTimeChart = compose(
  withOSRDData,
  withForcedEcoAllowanceSettings
)(SpaceTimeChart);

ce composant est utilisé pour stdcm

Donc il n'y a pas besoin de re-useSelector